cloudigrade/config/settings/base.py

```diff
@@ -94,7 +94,7 @@
 DATABASES = {
     'default': env.db('DATABASE_URL', default='postgres://localhost/cloudigrade'),
 }
-DATABASES['default']['ATOMIC_REQUESTS'] = True
+DATABASES['default']['ATOMIC_REQUESTS'] = env('DJANGO_ATOMIC_REQUESTS', default=True)
```

Comment:
Do you foresee us ever wanting to override this at the project-environment level?